docker/README.md

```diff
@@ -183,7 +183,7 @@ sudo apt-get -y install docker-ce docker-ce-cli containerd.io
 sudo systemctl disable apt-daily-upgrade.timer
 sudo systemctl stop apt-daily-upgrade.timer
 # Put our hostname in /etc/hosts otherwise sudo will give warnings
-echo 127.0.0.1 $(hostname) | sudo tee -a /etc/hosts
+echo 127.0.0.1 "$(hostname)" | sudo tee -a /etc/hosts
```

Comment:
I think this is overkill as it's only documentation and hostnames are not allowed to have whitespace.

Reply:
Removed it.